File: msgraph-mail/typescript/src/users/extension.ts
import {SerializationWriter, ParseNode, Parsable} from '@microsoft/kiota-abstractions';
import {Entity} from './entity';

export class Extension extends Entity implements Parsable<Extension> {
    /**
     * The deserialization information for the current model
     * @returns a Map<string, (item: Extension, node: ParseNode) => void>
     */
    public getFieldDeserializers () : Map<string, (item: Extension, node: ParseNode) => void> {
        return new Map<string, (item: Extension, node: ParseNode) => void>([...super.getFieldDeserializers(),
        ]);
    };
    /**
     * Serialiazes information the current object
     * @param writer Serialization writer to use to serialize this model
     * @returns a void
     */
    public serialize (writer: SerializationWriter) : void {
        super.serialize(writer);
    };
}

File: msgraph-mail/dotnet/Users/Extension.cs
using Kiota.Abstractions.Serialization;
using System;
using System.Collections.Generic;
using System.Linq;
namespace Graphdotnetv4.Users {
    public class Extension : Entity, IParsable<Extension> {
        /// <summary>The serialization information for the current model</summary>
        public new IDictionary<string, Action<Extension, IParseNode>> DeserializeFields => new Dictionary<string, Action<Extension, IParseNode>> {
        };
        /// <summary>
        /// Serialiazes information the current object
        /// <param name="writer">Serialization writer to use to serialize this model</param>
        /// </summary>
        public new void Serialize(ISerializationWriter writer) {
            base.Serialize(writer);
        }
    }
}
